src/mods/vr/D3D11Component.cpp

```diff
@@ -559,8 +559,9 @@ vr::EVRCompositorError D3D11Component::on_frame(VR* vr) {
                 (void*)m_left_eye_tex.Get(), vr::TextureType_DirectX, vr::ColorSpace_Auto,
                 submit_pose
             };
-
-            const auto e = vr::VRCompositor()->Submit(vr::Eye_Left, &left_eye, &vr->m_left_bounds, vr::EVRSubmitFlags::Submit_TextureWithPose);
+            auto const left_bounds = vr::VRTextureBounds_t{runtime->m_view_bounds[0][0], runtime->m_view_bounds[0][2],
```

Comment:
lots of uses of `auto const` that go against the project's style of `const auto`

Reply:
Will tidy that up